### PR TITLE
 Digital council identifier updates

### PR DESCRIPTION
**Pull request by Sgtpluck**
This PR:
* Updates the README (I struggled getting the project running until I realized that I was trying to use out-of-date instructions on the wrong branch)
* Adds 18F@gsa.gov as a contact as part of the identifier (and changes the text font, the default for the identifier on this project is not accessible with the gray background) 
* Adds a link to the privacy policy.

Thanks to @igorkorenfeld for doing the initial work [here](https://github.com/18F/brand/pull/280)

### CODE REVIEW

File: _includes/footer.html
Comment (igorkorenfeld) [inline _includes/footer.html]: I know on the other ones that I've worked on the footer has a class of `usa-footer` instead, and it contains the `usa-identifier`, and so I'm wondering it makes sense to do the same here? That way the footer can hold the identifier plus anything else it might need.

Comment (Sgtpluck) [reply _includes/footer.html]: that makes sense!

**Review by igorkorenfeld — CHANGES_REQUESTED** [review]
I think we've settled on the pieces and format of the footer now over in the [slack thread](https://gsa-tts.slack.com/archives/C02UQQWDSR1/p1655388659554619), so just a couple more adjustments:

- [ ] Changing the link text to match the format of the ones in the identifier on the [content guide](https://content-guide.18f.gov/), except taking out `Contact Us`

- [ ] Adding "This project is maintained by [18F](https://18f.gsa.gov/)" in the top footer section before "Have questions or would like to..."

**Comment by Sgtpluck** [comment]
@igorkorenfeld can you be a little more specific about the "This project is maintained by [18F](https://18f.gsa.gov/)" line? 
My understanding of that is it should look like this:
Screen Shot 2022-06-17 at 11 46 00 AM
so please clarify if that's not the case.

**Comment by igorkorenfeld** [comment]
With two lines works better than what I had in mind so yes the screenshot looks good!

**Comment by igorkorenfeld** [comment]
This is how I had it on the one I'm reworking. But I think the two line approach might be cleaner. What do you think?
image

**Comment by Sgtpluck** [comment]
i like the two lines personally (i had it as one initially and thought to myself, "this can't be what he meant" haha

**Comment by igorkorenfeld** [comment]
Hahah. I put in a "Very thin space" to help with the period, but yeah, let's go with 2 lines!

File: _includes/footer.html
Comment (igorkorenfeld) [inline _includes/footer.html]: Minor thing, I think you could pull in this link from the identifier data but either way works

**Review by igorkorenfeld — APPROVED** [review]
Looks good to me, but I'll let the guide maintainers have the final say

File: _includes/footer.html
Comment (igorkorenfeld) [inline _includes/footer.html]: Just noticed this one, same issue with the hyphen versus underscore on `privacy_policy_url`

**Review by igorkorenfeld — DISMISSED** [review]

File: _includes/footer.html
Comment (Sgtpluck) [reply _includes/footer.html]: oy, thanks, could have sworn i fixed it here!

**Review by myfriendgu — APPROVED** [review]
Had a call with Claire and Davida in which we reviewed.

**Pull request merged**
